sections/ads.txt

```diff
@@ -445,6 +445,7 @@ hazipatika.com##[class*="advert"]
 helyitipp.hu##[id*="hirdet"]
 heol.hu##[class*="hirdetes"]
 hetek.hu##[id*="popup"]
+www.hellovidek.hu##.kalkulator
```

Comment:
Actually, `www` is redundant here

What exactly does the `.kalkulator` selector block?

This site also has a floating recommender annoyance, as well as an article recommender pointing to an external site:

<details>

<summary>Show screenshot</summary>

![image](https://github.com/hufilter/hufilter-dev/assets/57285466/e7309785-6cc7-411e-8d9c-cb2aac2d372c)

</details>